app/app.html

```diff
@@ -15,14 +15,14 @@
       <!-- view: global_header/global_header -->
     </header>
     <div id="container" class="row">
-      <div class="row-inner">
+      <div class="row-inner-desktop-padded">
```

Comment:
Classes shouldn't contain any reference to desktop or mobile. This should all just be handled with the breakpoint code.